src/ui.tsx

```diff
@@ -423,7 +439,7 @@ function PlanPreview(
           <path
             key={i}
             d={line.reduce((m, {x, y}, j) => m + `${j === 0 ? "M" : "L"}${x} ${y}`, "")}
-            style={i % 2 === 0 ? {stroke: "rgba(0, 0, 0, 0.3)", strokeWidth: 0.5} : {}}
+            style={i % 2 === 0 ? {stroke: "rgba(0, 0, 0, 0.3)", strokeWidth: 0.5} : { stroke: "rgba(0, 0, 0, 0.8)", strokeWidth: state.planOptions.penStrokeWidth * Device.Axidraw.stepsPerMm }}
```

Comment:
You probably know this but I just understood why it has to be multiplied by `Device.Axidraw.stepsPerMm`. It's because it compensates for the ```transform={`scale(${1 / Device.Axidraw.stepsPerMm})`}>``` further up. Nevermind. 😄